scripts/flatten.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-multi-str */
+
+const exec = require("child_process").exec
+const fs = require("fs")
+const os = require("os")
+const path = require("path")
+
+if (process.argv.length < 3) {
+  console.log("\n\
+    Usage: yarn flatten </path/to/contracts/folder/ | /path/to/contract/file.sol>\n\
+       or: npm run flatten </path/to/contracts/folder/ | /path/to/contract/file.sol>\n\n\
+  ")
+  process.exit(0)
+}
+
+try {
+  createFolder("./flattened")
+  const stats = fs.lstatSync(process.argv[2])
+  if (stats.isFile()) {
+    flatten(path.parse(process.argv[2]).dir, process.argv[2])
+  } else if (stats.isDirectory()) {
+    const basedir = path.normalize(process.argv[2]).replace(/\\/g, "/")
+    const files = fs.readdirSync(basedir).filter(filename => filename.endsWith(".sol"))
+    files.forEach(filename => {
+      flatten(basedir, filename)
+    })
+    console.log(`Processed ${files.length} Solidity files.`)
+  }
+} catch (e) {
+  console.log("Fatal:", e)
```

Comment:
Errors should be logged as such.
```suggestion
  console.error("Fatal:", e)
```